templates/page/reset-button.php

```diff
@@ -0,0 +1,5 @@
+<?php
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+?>
+<a data-component="reset" class="hp-link"><?php esc_html_e( 'Reset Filters', 'hivepress' ) ?></a>
```

Comment:
Maybe let's make it a container or content block? Seems to be too vague to require a template part, and it'll appear in the vendor, request, etc. filter forms.